src/index.js

```diff
@@ -3,22 +3,13 @@ import defaultTheme from "@kiwicom/orbit-design-tokens";
 import { withTheme } from "theming";
 
 import * as Icons from "./icons";
+import * as DeprecatedComponents from "./_deprecated";
 
 export { default as Button } from "./Button";
 export { default as CarrierLogo } from "./CarrierLogo";
 export { default as Heading } from "./Heading";
 export { default as Text } from "./Text";
 export { default as Icon } from "./Icon";
-export { default as InputText } from "./InputText";
-export { default as InputTextarea } from "./InputTextarea";
-export { default as Checkbox } from "./Checkbox";
-export { default as Header } from "./Container/Header";
-export { default as Section } from "./Container/Section";
-export { default as Select } from "./Select";
-export { default as SystemMessage } from "./SystemMessage";
-export { default as FieldFeedback } from "./FieldFeedback";
 export { default as TextLink } from "./TextLink";
-export { default as Typography } from "./Typography";
-export { default as Radio } from "./Radio";
 export { default as ThemeProvider } from "./Theming/ThemeProvider";
-export { Icons, defaultTheme, withTheme };
+export { Icons, DeprecatedComponents, defaultTheme, withTheme };
```

Comment:
This is breaking change ⚠️ 

I mean - I'm totally ok to move these components to deprecated folder, but it's nonwelcomed for me as a user of orbit to change imports all over the app if I want to bump the version.

If I currently use `Typography`, the goal is to get rid of using it in my code over time and replace it with new components => There might be some time when I use both deprecated and new components in code

With this change, I would be stuck with `0.0.0-rc7` because I don't want to change imports for components I plan to remove anyway.

If we intend to hint users these components should not be used, we should put `console.warn` or `console.error` into them.

TL;DR: Don't change how components can be imported.